lib/ex_unit/test/ex_unit/diff_test.exs

```diff
@@ -125,6 +125,39 @@ defmodule ExUnit.DiffTest do
     assert script(list1, list2) == expected
 
     assert script([], []) == [eq: "[]"]
+
```

Comment:
Let's create a new test case especifically about testing lists that are subsets instead of adding more examples to those already large cases :)